### ISSUE TITLE
siteWidth problem

### ISSUE THREAD

**Issue by coolzilj** [anjlab/bootstrap-rails#17]
According to http://twitter.github.com/bootstrap/scaffolding.html,
`$siteWidth` counts number of columns and gutters to set width of the .container-fixed() mixin,
and I set `$siteWidth` to`970px`,but it didn't change the container size, it still be `940px`
Here is my code in bootstrap.scss:
```@import "variables.scss"; // Modify this for custom colors, font-sizes, etc
$gridGutterWidth:10px;
$gridColumnWidth:70px;
$siteWidth:970px;
@import "mixins.scss";
...

```
How can I change the container size correspondingly?
```

**Comment by yury** [comment]
Thanks for issue.
1. $siteWidth - is error in docs. This var is called $gridRowWidth now and it is computed (`($gridColumns * $gridColumnWidth) + ($gridGutterWidth * ($gridColumns - 1))`). So just use $gridGutterWidth and $gridColumnWidth
2. There was bug with variable overrides. It is fixed in v2.0.0.3, please update.
3. Steps to set new .container width:
   - in application.css replace `*= require bootstrap` with `*= require overrides`
   - create `overrides.css.scss`
   - in overrides.css.scss place:

```
$gridColumnWidth: 70px;
$gridGutterWidth: 10px;

@import "bootstrap"
```

**Comment by coolzilj** [comment]
Thanks for the fix.It's alright now.
